src/applications/check-in/pages/CheckIn/DisplayMultipleAppointments.jsx

```diff
@@ -44,6 +50,15 @@ export default function DisplayMultipleAppointments(props) {
           );
         })}
       </ol>
+      <p data-testid="update-text">
+        <strong>Latest update:</strong>{' '}
+        {format(new Date(), 'MMMM dd, yyyy HH:mm')}
```

Comment:
Does this also refresh when the button is clicked?

Reply:
yep! 😃